src/AppBanner/index.tsx

```diff
@@ -57,36 +52,24 @@ export const AppBanner = (props: Props): ReactElement | null => {
 
     const [showBanner, setShowBanner] = useState(true);
 
-    const onCloseClick = (
-        evt: React.MouseEvent<HTMLButtonElement, MouseEvent>,
-        internalButtonId: number,
-    ): void => {
-        evt.preventDefault();
+    const logToBrazeAndOphan = (internalButtonId: number): void => {
+        logBannerCloseToOphan(
```

Comment:
I wonder if this should be slightly renamed or reorganized? I think it's potentially bit confusing that the `logBannerCloseToOphan` function takes the `logButtonClickWithBraze` function as an argument (i.e. the scope is greater than the function name suggests, since it also reports to Braze). What do you reckon?

I think perhaps there's also an extra level of function application here that we don't need as `logToBrazeAndOphan` invokes `logBannerCloseToOphan` but doesn't do anything else. Perhaps the imported function could be renamed to something more general like `logClickToBrazeAndOphan` and invoked directly by `onCloseAction` etc?

Reply:
This is a bit embarrassing ... I refactored all this out (in Part 4, when adding the new newsletter banner) after I discovered we already have the code to handle it in the `utils/tracking.ts` file. I have no idea why banners weren't already using that code - insights on any edge case I'm missing would be very welcome!

Reply:
Ahh interesting. From the commit which last touched `utils/tracking.ts`:

```
This includes a refactoring to expose a single trackClick callback to
the individual epic components, instead of submitComponentEvent and
logButtonClickWithBraze separately. This keeps the underlying code
simpler and also DRYs things up a bit. I've only done this refactoring
for epics here. Banners will be done in a separate PR.
```

I guess this bit never happened - "Banners will be done in a separate PR". So it'd be good to make banners and epics consistent. Let me know if I can help with that.